Depth-Anything-V2-list3/depth_anything_v2/dinov2_layers/attention.py

```diff
@@ -8,24 +8,11 @@
 #   https://github.com/facebookresearch/dino/blob/master/vision_transformer.py
 #   https://github.com/rwightman/pytorch-image-models/tree/master/timm/models/vision_transformer.py
 
-import logging
 
 from torch import Tensor
 from torch import nn
 
 
-logger = logging.getLogger("dinov2")
-
-
-try:
-    from xformers.ops import memory_efficient_attention, unbind, fmha
-
-    XFORMERS_AVAILABLE = True
-except ImportError:
-    logger.warning("xFormers not available")
-    XFORMERS_AVAILABLE = False
```

Comment:
@dan-jfisher I currently disabled [xformers](https://github.com/facebookresearch/xformers). Should we add a ticket to look into it at some point to evaluate whether we get performance gains when using it?

Reply:
Did you disable it because it wasn't scriptable?

Reply:
Requires some refactoring in two files to make it TorchScript friendly for export. So I had initially commented it out to get the basic version running. Oddly, the authors also did not include `xformers` in their dependency list.